angular.json

```diff
@@ -3806,1597 +3806,1597 @@
                 }
```

Comment:
I think this file, `ng.json` and `tsconfig.base.json` should not be changed in the scope of that pr.

Reply:
> I think this file, ng.json and tsconfig.base.json should not be changed in the scope of that pr.

Bump for this one.